types/fetch-mock/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "strict-export-declare-modifiers": false
```

Comment:
I don't see any failures from `strict-export-declare-modifiers` when I remove the ignore. Could you try reinstalling your `node_modules` to see if you still have any failures?